Testing/VersionMinerTests/ActionInputsTests.cs

```diff
@@ -34,6 +34,11 @@ public void Ctor_WhenConstructed_PropsHaveCorrectDefaultValuesAndDecoratedWithAt
         inputs.GetAttrFromProp<OptionAttribute>(nameof(ActionInputs.RepoName))
             .AssertOptionAttrProps("repo-name", true, "Gets or sets the name of the repository.");
 
+        inputs.RepoToken.Should().BeEmpty();
+        typeof(ActionInputs).GetProperty(nameof(ActionInputs.RepoToken)).Should().BeDecoratedWith<OptionAttribute>();
+        inputs.GetAttrFromProp<OptionAttribute>(nameof(ActionInputs.RepoToken))
+            .AssertOptionAttrProps("repo-token", false, "The PAT used to authenticate to the repository.");
```

Comment:
Update the unit test to reflect the change in the `ActionInputs` class

```suggestion
            .AssertOptionAttrProps("repo-token", false, "The GitHub or PAT token used to authenticate to the repository.");
```